packages/react-charting/src/components/DonutChart/DonutChart.base.tsx

```diff
@@ -313,4 +329,22 @@ export class DonutChartBase extends React.Component<IDonutChartProps, IDonutChar
   private _getHighlightedLegend() {
     return this.state.selectedLegend || this.state.activeLegend;
   }
+
+  private _showTooltip = (text: string | number, evt: any) => {
+    this._tooltip = document.getElementById(this._tooltipId);
+    if (this._tooltip) {
+      this._tooltip!.innerHTML = text.toString();
+      this._tooltip!.style.display = 'block';
```

Comment:
Can this be done using react states and not direct dom interactions

Reply:
using react states require the title of the rect to be set in which case, there is no way to override the style of the tooltip as it appears as follows:
<img width="253" alt="tooltip_example" src="https://github.com/microsoft/fluentui/assets/120183316/88754b18-7441-4507-9855-58f01847965c">
As a result, I have created a ref to the tooltip element as used it instead of querying the dom.